### PR TITLE
INDYAV: added directories and initial documentation.

### PR DESCRIPTION
**Pull request by popeyedpanther**

### CODE REVIEW

File: IndyAV/control/README.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Control
+Software that relates to generating vehicle actuation commands should go here. Possible that path planning will go here if the algorithm is tightly coupled with the control algorithm.
```
Comment (MarshallRawson) [inline IndyAV/control/README.md]:
```suggestion
Software that relates to generating vehicle actuation commands should go here. Possible that path planning will go here if the algorithm is tightly coupled with the control algorithm.
```

File: IndyAV/localization/README.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
+# Localization
+Software that determines the ego vehicles state given interial and extroperceptive sensors.
+
+## Packages
+Information about current included software will go here.
```
Comment (MarshallRawson) [inline IndyAV/localization/README.md]: See previous comment

File: IndyAV/perception/README.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
+# Perception
+This directory should contain code that uses sensors to classify entities in the world.
+
+## Packages
+Information about current included software will go here.
```
Comment (MarshallRawson) [inline IndyAV/perception/README.md]: See previous comment

File: IndyAV/planning/README.md [hunk 4]
```diff
@@ -0,0 +1,7 @@
+# Planning
+Software that relates to decision making and path planning should go here.
```
Comment (MarshallRawson) [inline IndyAV/planning/README.md]: See previous comment

File: IndyAV/simulation/README.md [hunk 5]
```diff
@@ -0,0 +1,8 @@
+# Simulation
+Vehicle and world models should go here. Along with any code that is needed for simulation to work.
```
Comment (MarshallRawson) [inline IndyAV/simulation/README.md]: See previous comment

File: docs/indyav/hardware/go_kart/electronics.rst [hunk 6]
```diff
@@ -0,0 +1,3 @@
+Electronics Documentation
+=========================
+Power electronics, computation hardware should go here
```
Comment (MarshallRawson) [inline docs/indyav/hardware/go_kart/electronics.rst]: WOOOOOOOOOOOO HARDWARE DOCUMENTATION IN THE MONO REPO!!!!!

File: docs/indyav/hardware/go_kart/mechanical.rst [hunk 7]
```diff
@@ -0,0 +1,4 @@
+Mechanical Documentation
+========================
+Not sure if this is needed or what would go here. Maybe part drawings
```
Comment (MarshallRawson) [inline docs/indyav/hardware/go_kart/mechanical.rst]: Maybe info/links about the grab cad should go here

File: docs/indyav/onboarding.rst [hunk 8]
```diff
@@ -0,0 +1,26 @@
+Onboarding
```
Comment (MarshallRawson) [inline docs/indyav/onboarding.rst]: Onboarding -> Getting Started on Indy AV (really just not Onboarding)

**Comment by popeyedpanther** [comment]
Text has been updated and the software component documentation has been moved to /docs as .rst files.

File: docs/indyav/index.rst [hunk 9]
```diff
@@ -0,0 +1,30 @@
...
+Welcome to team Gator Double Dragon's documentation. The team is a collabortaion
+between the University of Florida and Kookmin University for the
+`Indy Autonomous Challenge <https://www.indyautonomouschallenge.com/>`_.
+
+.. image:: gator_double_dragon.jpg
+   :width: 600px
+   :align: center
+   :height: 420px
+
+Collaborators
```
Comment (MarshallRawson) [inline docs/indyav/index.rst]: `Collaborators` -> `Collaborating Laboratories`

File: docs/indyav/onboarding.rst [hunk 10]
```diff
@@ -0,0 +1,26 @@
+Getting Started on Indy AV
+==========================
+
```
Comment (MarshallRawson) [inline docs/indyav/onboarding.rst]: add ``Make sure to checkout `Development <../development/index.html>`_ first.``

File: docs/indyav/onboarding.rst [hunk 11]
```diff
@@ -0,0 +1,26 @@
...
+Collaboration
```
Comment (MarshallRawson) [inline docs/indyav/onboarding.rst]: `Collaboration` -> `Collaborating on Indy AV`

File: docs/indyav/onboarding.rst [hunk 12]
```diff
@@ -0,0 +1,26 @@
...
+-------------
+
+Slack
+*****
+Join the slack to keep updated on the developement at `uf-cimar <uf-cimar.slack.com>`_.
```
Comment (MarshallRawson) [inline docs/indyav/onboarding.rst]: ``Join the slack to keep updated on the developement at `uf-cimar <uf-cimar.slack.com>`_.`` -> ``Join the slack to keep updated on the developement at `uf-cimar <https://uf-cimar.slack.com>`_.``

File: docs/indyav/onboarding.rst [hunk 13]
```diff
@@ -0,0 +1,26 @@
...
+Any valid @ufl.edu email will be able to create an account.
+
+Github
+******
+If you see this documentation you are probably already aware of the repository
```
Comment (MarshallRawson) [inline docs/indyav/onboarding.rst]: ``If you see this documentation you are probably already aware of the repository`` -> ``All work (except CAD) and documentation for Indy AV will go in the `MIL REPO <https://github.com/uf-mil/mil>`_.``

File: docs/indyav/onboarding.rst [hunk 14]
```diff
@@ -0,0 +1,26 @@
...
+
+Grabcad
+*******
+We use GrabCAD Workbench to manage mechanical design files.
+
+Software
+--------
+We will follow MIL's `current development practices <https://github.com/uf-mil/mil/blob/master/docs/development/development_guide.md>`_
```
Comment (MarshallRawson) [inline docs/indyav/onboarding.rst]: ``We will follow MIL's `current development practices <https://github.com/uf-mil/mil/blob/master/docs/development/development_guide.md>`_`` -> ``We will follow `MIL's development practices <../development/development_guide.html>`_.``

File: docs/indyav/software/software.rst [hunk 15]
```diff
@@ -0,0 +1,19 @@
...
+Middleware
+----------
+We are currently using ROS for its' ease of developement. Which allows us to be
+able to prototype early algorithms. It is possible that this will change during
+the course of the competition as more real time requirements are encountered
+because of the speed of the vehicle.
+
+.. toctree::
+   :maxdepth: 1
+   :caption: Elements of Autonomony
```
Comment (MarshallRawson) [inline docs/indyav/software/software.rst]: `Elements of Autonomony` -> `Core Software Systems`

File: docs/indyav/onboarding.rst [hunk 16]
```diff
@@ -0,0 +1,26 @@
...
+Join the slack to keep updated on the developement at `uf-cimar <uf-cimar.slack.com>`_.
+Any valid @ufl.edu email will be able to create an account.
+
+Github
+******
+If you see this documentation you are probably already aware of the repository
+
+Grabcad
+*******
+We use GrabCAD Workbench to manage mechanical design files.
```
Comment (MarshallRawson) [inline docs/indyav/onboarding.rst]: Add links to the Grab CAD

File: docs/indyav/hardware/hardware.rst [hunk 17]
```diff
@@ -0,0 +1,16 @@
...
+.. toctree::
+   :maxdepth: 1
+
+   Electronics <go_kart/electronics>
+   Mechatronics <go_kart/mechatronics>
+   Mechanical <go_kart/mechanical>
+
+Indy Light
+----------
+TBD
```
Comment (MarshallRawson) [inline docs/indyav/hardware/hardware.rst]: Does the Indy light car specs exist?

File: docs/indyav/software/simulation.rst [hunk 18]
```diff
@@ -0,0 +1,11 @@
+Simulation
+==========
+Vehicle and world models should go here. Along with any code that is needed for simulation to work.
```
Comment (MarshallRawson) [inline docs/indyav/software/simulation.rst]: Remove `Vehicle and world models should go here. Along with any code that is needed for simulation to work.`

File: docs/indyav/hardware/go_kart/electronics.rst (hunk 6 above)
Comment (MarshallRawson) [inline docs/indyav/hardware/go_kart/electronics.rst]: `Power electronics, computation hardware should go here` -> `TODO`

File: docs/indyav/hardware/go_kart/mechanical.rst (hunk 7 above)
Comment (MarshallRawson) [inline docs/indyav/hardware/go_kart/mechanical.rst]: `Not sure if this is needed or what would go here. Maybe part drawings` -> `TODO`

File: docs/indyav/software/control.rst [hunk 19]
```diff
@@ -0,0 +1,24 @@
...
+=======
+
+Control Packages
+----------------
+No software developed yet.
+
+Discussion
+-------------
+Initial development will start simple.
+
```
Comment (MarshallRawson) [inline docs/indyav/software/control.rst]: Remove lines 4 - 11

File: docs/indyav/software/planning.rst [hunk 20]
```diff
@@ -0,0 +1,9 @@
+Planning
+========
+
+Planning Packages
+-----------------
+No software developed yet.
+
+Discussion
+----------
```
Comment (MarshallRawson) [inline docs/indyav/software/planning.rst]: Remove lines 7-9

File: docs/indyav/hardware/hardware.rst (hunk 17 above)
Comment (popeyedpanther) [reply docs/indyav/hardware/hardware.rst]: No

Comment (MarshallRawson) [reply docs/indyav/hardware/hardware.rst]: cool

File: docs/indyav/index.rst (hunk 9 above)
Comment (popeyedpanther) [reply docs/indyav/index.rst]: I don't believe this change provides any benefit

File: docs/indyav/software/control.rst (hunk 19 above)
Comment (popeyedpanther) [reply docs/indyav/software/control.rst]: The headers define the template. I do not think they should be removed.

File: docs/indyav/software/planning.rst (hunk 20 above)
Comment (popeyedpanther) [reply docs/indyav/software/planning.rst]: see previous comment

**Pull request merged**